### PR TITLE
add Flower Framework

### PR DESCRIPTION
**Pull request by mariaboerner1987**
Flower is a framework to easily transform your ML workload federated.
Added webpage of Flower and the corresponding paper. Also added the new Flower Blog.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Federated learning research is booming. We organize the papers by their targetin
 
 ### Blogs
 
+* Flower: A Friendly Federated Learning Framework [[Flower Blog]](https://flower.dev/blog/)
```
Comment (weimingwill) [inline README.md:47]: This "Blogs" section is mainly for the general introduction of federated learning. It is not very suitable to include them here. Could you remove this part? The other part is cool.

File: areas/system.md [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 ## Frameworks
 
+* Flower [[Web]](https://flower.dev/)
+  * Flower: A Friendly Federated Learning Framework [[Paper]](https://arxiv.org/abs/2007.14390)
```
Comment (weimingwill) [inline areas/system.md:15]: I guess you can also add it to the "## System Design" section.

File: README.md (hunk 1 above)
Comment (mariaboerner1987) [reply README.md]: done

File: areas/system.md (hunk 2 above)
Comment (mariaboerner1987) [reply areas/system.md]: I added the paper to system design and left the webpage of flower at the framework area.

**Pull request merged**